draft-ounsworth-pq-composite-sigs.mkd

```diff
@@ -323,6 +341,26 @@ Where each BIT STRING within the SEQUENCE is a signature value produced by one o
 
 The choice of `SEQUENCE OF BIT STRING`, rather than for example a single BIT STRING containing the concatenated signature values, is to gracefully handle variable-length signature values by taking advantage of ASN.1's build-in length fields.
 
+
+### Explicit Composite Signature
+
+The ASN.1 algorithm object for an explicit composite signature is:
+
+~~~ asn.1
+sa-CompositeSignature SIGNATURE-ALGORITHM ::= {
+    IDENTIFIER id-alg-explicit
+    VALUE CompositeSignatureValue
+    PARAMS ARE optional
+    PUBLIC-KEYS { pk-Composite }
+    SMIME-CAPS { IDENTIFIED BY id-alg-explicit } }
+}
+~~~
+
+In this case, id-alg-explicit is an algorithm and OID defined elsewhere which explicitly defines the component algorithms. PARAMS are optional as they will likely carry redundant information to id-alg-explicit. The definition of id-alg-explicit needs to specify the contents of VALUE, PARAMS, and SMIME-CAPS.
```

Comment:
Make clear that id-alg-explicit is a placeholder, and that it needs to specify a signature algorithm.